src/Bicep.Core/Diagnostics/DiagnosticBuilder.cs

```diff
@@ -130,6 +130,7 @@ private static string ToQuotedStringWithCaseInsensitiveOrdering(IEnumerable<stri
                 "BCP013",
                 "Expected a parameter identifier at this location.");
 
+            [Obsolete]
```

Comment:
Feel free to delete these if they're no longer used - we don't reuse error codes.